servicenow/stackstate_checks/servicenow/servicenow.py

```diff
@@ -57,6 +62,27 @@ class ChangeRequest(Model):
     assigned_to = ModelType(WrapperStringType)
 
 
+class ConfigurationItem(Model):
+    name = ModelType(WrapperStringType, required=True)
+    sys_class_name = ModelType(WrapperStringType, required=True)
+    sys_id = ModelType(WrapperStringType, required=True)
+    sys_tags = ModelType(WrapperStringType, default=WrapperStringType())
+    fqdn = ModelType(WrapperStringType, default=WrapperStringType())
+    host_name = ModelType(WrapperStringType, default=WrapperStringType())
+
+
+class CIRelation(Model):
+    sys_id = ModelType(WrapperStringType)
```

Comment:
Is `sys_id` required?

Reply:
nope it's optional. we just need `parent id` and `child id`. We don't have any concept for `external_id` right now for relation. so not required.